src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -91,30 +91,69 @@ public enum JvmSettings {
 
     // PERSISTENT IDENTIFIER SETTINGS
     SCOPE_PID(PREFIX, "pid"),
-    
-    // PROVIDER EZID (legacy) - these settings were formerly kept together with DataCite ones
-    SCOPE_PID_EZID(SCOPE_PID, "ezid"),
-    EZID_API_URL(SCOPE_PID_EZID, "api-url", "doi.baseurlstring"),
-    EZID_USERNAME(SCOPE_PID_EZID, "username", "doi.username"),
-    EZID_PASSWORD(SCOPE_PID_EZID, "password", "doi.password"),
+    PID_PROVIDERS(SCOPE_PID, "providers"),
+    PID_DEFAULT_PROVIDER(SCOPE_PID, "default-provider"),
+    PROVIDERS_DIRECTORY(SCOPE_PID, "provider-implementations-directory"),
```

Comment:
For exporters, we created `dataverse.spi.exporters.directory`. Shouldn't we use a similar scope und `spi` for PID providers, too?